src/chainstate/stacks/db/blocks.rs

```diff
@@ -4628,7 +4630,7 @@ impl StacksChainState {
                 tx.run_contract_call(
                     &sender.into(),
                     None,
-                    &boot_code_id("pox", mainnet),
+                    &boot_code_id(active_pox_contract, mainnet),
```

Comment:
Lol this fixes #3171 :)

Reply:
Didn't realize this was already an issue!